src/server/ServerMethod.js

```diff
@@ -87,6 +87,7 @@ export default class ServerMethod {
             path: url,
             params: urlParams
           },
+          postBody: args,
```

Comment:
yeah, args (if it's coming from req.body, not req.body.args) is very much an object... I'll push my test version in a moment which _works_ with the changes in here...